src/server/game/Spells/SpellMgr.cpp

```diff
@@ -3437,6 +3437,9 @@ void SpellMgr::LoadSpellInfoCorrections()
                 // unsummoned when spell missile hits the ground so nothing happen in result
                 spellInfo->DurationEntry = sSpellDurationStore.LookupEntry(85);
                 break;
+            case 66240: //Anubarak Leeching Swarm damage
+                spellInfo->AttributesEx3 |= SPELL_ATTR3_ONLY_TARGET_PLAYERS;
```

Comment:
yeap, but all spellmgr.cpp is hack no?! ._.
btw, is best left bugged or fix it with spellmgr ?!
same for spells with (channel while moving) and scripts remove unit casting flag...